exasol_bucketfs_utils_python/bucketfs_connection_config.py

```diff
@@ -8,14 +8,16 @@ class BucketFSConnectionConfig:
     """
 
     @typechecked(always=True)
-    def __init__(self, host: str, port: int, user: str, pwd: str, is_https=False):
+    def __init__(self, host: str, port: int,
+                 user: str, pwd: str, is_https=False):
```

Comment:
is_https has no typehint
```suggestion
                 user: str, pwd: str, is_https: str =False):
```

Reply:
thx, it is boolean.